source/type/pb_converter.cpp

```diff
@@ -856,12 +856,25 @@ void PbCreator::Filler::fill_pb_object(const nt::VehicleJourney* vj, pbnavitia::
         const auto& vector_bp = navitia::vptranslator::translate(*vj->base_validity_pattern());
 
         fill(vector_bp, vehicle_journey->mutable_calendars());
+
+        if (!pair_jp.second.freq_vjs.empty()) {
+            for (const auto& frequency_vehicle_journey : pair_jp.second.freq_vjs) {
```

Comment:
this seems overly complicated:  `FrequencyVehicleJourney` inherit of `VehicleJourney` so you can just `dynamic_cast` after having checked that it's a freq vj